.github/workflows/run-zombienet-tests.yml

```diff
@@ -133,23 +161,24 @@ jobs:
         with:
           name: binaries
           path: binaries
-
-  # Run each test as a separate job using a matrix strategy
   run-tests:
     runs-on: self-hosted
     needs: [ "get-tests", "build" ]
     env:
       CARGO_TARGET_DIR: "target"
     strategy:
       fail-fast: false
-      matrix: ${{ fromJSON(needs.get-tests.outputs.matrix) }}
+      matrix: ${{ fromJson(needs.get-tests.outputs.matrix) }}
```

Comment:
I only see `fromJSON` in docs? Why this change?

https://docs.github.com/en/actions/reference/workflows-and-actions/expressions#fromjson

Reply:
It seems like both options are possible. I've just made it consistent with other similar cases in our repo.